grafana-deployment/terraform.tfstate

```diff
@@ -0,0 +1,187 @@
+          }
+        }
+      ]
+    },
+    {
+      "mode": "managed",
+      "type": "aws_instance",
+      "name": "grafana-server",
+      "provider": "provider.aws",
+      "instances": [
+        {
+          "schema_version": 1,
+          "attributes": {
+            "ami": "ami-056ee704806822732",
+            "arn": "arn:aws:ec2:us-west-1:656503231766:instance/i-0934c8c5773f6a5ea",
+            "associate_public_ip_address": true,
+            "availability_zone": "us-west-1b",
+            "cpu_core_count": 1,
+            "cpu_threads_per_core": 2,
+            "credit_specification": [],
+            "disable_api_termination": false,
+            "ebs_block_device": [],
+            "ebs_optimized": false,
+            "ephemeral_block_device": [],
+            "get_password_data": false,
+            "host_id": null,
+            "iam_instance_profile": "",
+            "id": "i-0934c8c5773f6a5ea",
+            "instance_initiated_shutdown_behavior": null,
+            "instance_state": "running",
+            "instance_type": "m5d.large",
+            "ipv6_address_count": 0,
+            "ipv6_addresses": [],
+            "key_name": "california-key-benchmark",
+            "monitoring": false,
+            "network_interface": [],
+            "network_interface_id": null,
+            "password_data": "",
+            "placement_group": "",
+            "primary_network_interface_id": "eni-0eb0ff42216138c72",
+            "private_dns": "ip-172-31-1-169.us-west-1.compute.internal",
+            "private_ip": "172.31.1.169",
+            "public_dns": "ec2-54-183-123-11.us-west-1.compute.amazonaws.com",
+            "public_ip": "54.183.123.11",
+            "root_block_device": [
+              {
+                "delete_on_termination": true,
+                "encrypted": false,
+                "iops": 600,
+                "kms_key_id": "",
+                "volume_id": "vol-0956a934d5cac026a",
+                "volume_size": 200,
+                "volume_type": "gp2"
+              }
+            ],
+            "security_groups": [
+              "hmy.pub.website"
+            ],
+            "source_dest_check": true,
+            "subnet_id": "subnet-551f560e",
+            "tags": {
+              "Name": "Grafana-tf",
+              "Project": "Harmony"
+            },
+            "tenancy": "default",
+            "timeouts": null,
+            "user_data": "502eb990b842800431ba487f4de6882d6444e452",
+            "user_data_base64": null,
+            "volume_tags": {
+              "Name": "HarmonyNode-monitoring-Volume",
+              "Project": "Harmony"
+            },
+            "vpc_security_group_ids": [
+              "sg-04bd70e941e53e557"
+            ]
+          },
+          "private": "eyJlMmJmYjczMC1lY2FhLTExZTYtOGY4OC0zNDM2M2JjN2M0YzAiOnsiY3JlYXRlIjo2MDAwMDAwMDAwMDAsImRlbGV0ZSI6MTIwMDAwMDAwMDAwMCwidXBkYXRlIjo2MDAwMDAwMDAwMDB9LCJzY2hlbWFfdmVyc2lvbiI6IjEifQ==",
```

Comment:
you sure you want to check in this tfstate file?  what's this private?

Reply:
.tfstate file is used to keep tracking the instance status on the cloud. Adding it to .gitignore too.